database/liquibase/data/shersched.db.changelog-courthousejustindata.xml

```diff
@@ -0,0 +1,726 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog 
+	  xmlns="http://www.liquibase.org/xml/ns/dbchangelog" 
+	  xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" 
+	  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+	  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd"
+	  context="prod">
```

Comment:
Does this changeset make sense for dev as well?  Should we remove the prod context so it's run regardless?

Reply:
Yes - good catch, and I'm going to find my template and remove the prod context from there as well.  Will update and repush shortly